plugins/tiledb/io/TileDBWriter.cpp

```diff
@@ -367,16 +367,18 @@ void TileDBWriter::ready(pdal::BasePointTable &table)
 #if TILEDB_VERSION_MAJOR >= 2
     #if ((TILEDB_VERSION_MINOR > 1) || (TILEDB_VERSION_MAJOR > 2))
         else
+        {
             domain.add_dimension(tiledb::Dimension::create<double>(*m_ctx, "X",
                 {{dimMin, dimMax}}))
                 .add_dimension(tiledb::Dimension::create<double>(*m_ctx, "Y",
                 {{dimMin, dimMax}}))
                 .add_dimension(tiledb::Dimension::create<double>(*m_ctx, "Z",
                 {{dimMin, dimMax}}));
+            m_schema->set_cell_order(TILEDB_HILBERT);
+        }
     #endif
 #endif
-        m_schema->set_domain(domain).set_order(
-            {{TILEDB_ROW_MAJOR, TILEDB_ROW_MAJOR}});
+        m_schema->set_domain(domain);
```

Comment:
Does this remove the ordering for TILEDB version < 2? Does this matter?

Reply:
No, it uses the default ordering for TileDB version < 2.